### PR TITLE
Enhancement: Set charset for all files to UTF-8

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] sets the charset for all files to `UTF-8`

Follows #554.

💁‍♂️ I would not mind pulling up more attributes!

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,10 +1,10 @@
 root = true
 
 [*]
+charset = utf-8
 insert_final_newline = true
 
 [*.yaml]
-charset = utf-8
 indent_size = 2
 indent_style = space
```
Comment (localheinz) [inline .editorconfig:9]: Still using mixed indentation in PHP files, but could be easily fixed after merging #559 and enabling the [`indentation_type`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/doc/rules/whitespace/indentation_type.rst) fixer.

**Comment by cmb69** [comment]
Hmm, are we sure that all files are UTF-8 encoded? What about binary files (e.g. some images)?

**Comment by localheinz** [comment]
@cmb69 

> Hmm, are we sure that all files are UTF-8 encoded? What about binary files (e.g. some images)?

I am not sure. 

> PS: well, binary files shouldn't be an issue.

👍

**Review by cmb69 — APPROVED** [review]
Looks like most, if not all, files are already UTF-8 encoded, and any accidential changes should be caught when reviewing the commit, so let's merge this.

Thank you for the RP!

**Pull request merged**